src/ecies/ecies.ts

```diff
@@ -0,0 +1,30 @@
+import init from "../pkg/ecies/cloudproof_ecies"
```

Comment:
call this init.ts as in aes256gcm ? or the other way around, what's more homogeneous with the other functions :)